packages/circuit-ui/components/Popover/__snapshots__/Popover.spec.tsx.snap

```diff
@@ -223,19 +223,16 @@ exports[`Popover styles should render popover on auto 1`] = `
         <svg
           class="circuit-2"
           fill="none"
-          height="16"
-          viewBox="0 0 16 16"
-          width="16"
+          height="24"
+          size="24"
```

Comment:
Example of the `size` making it into the DOM. Will need to investigate a bit further, because this doesn't happen consistently, and I think that we're destructuring `size` as expected in the icon components (will add another comment on the relevant spot below 👇)